### PR TITLE
commentd code

### PR DESCRIPTION
**Pull request by vidis01**
commentd code

### CODE REVIEW

**Review by kulverstux — COMMENTED** [review]
needs more commenting

**Review by kulverstux — CHANGES_REQUESTED** [review]
need better text formatting

**Comment by vidis01** [comment]
All good, mis understanding.

File: Learning/Program.cs [hunk 1]
```diff
@@ -130,9 +130,9 @@ static void Main(string[] args)
             #endregion
 
 
-            var textAnalyzer = new TextAnalyzer();
-            textAnalyzer.AnalyzedText("TestFile.txt");
-            textAnalyzer.MainMenuChoice();            
+            //var textAnalyzer = new TextAnalyzer();
+            //textAnalyzer.AnalyzedText("TestFile.txt");
+            //textAnalyzer.MainMenuChoice();
```
[the remark this answers is not in the document]
Comment (vidis01) [reply Learning/Program.cs:135]: ne, viskas ok

**Review by kulverstux — APPROVED** [review]
looks good

**Review by lagrinius — APPROVED** [review]

**Pull request merged**